style2.css

```diff
@@ -0,0 +1,287 @@
+body {
+    background-color: rgb(211, 211, 211);
+text-align: center;
+}
+
+.workspace {
+    position: relative;
+margin-top: auto;
+margin-left: auto;
+margin-right: auto;
+margin-bottom: auto; 
+  width: 1024px;
+  height: 765px;
+   background-color: white;
+
+}
+
+.color-palet {
+    position: relative;
+    text-align: center;
+    width: 350px;
+    top: 100px;

```

Comment:
The position of an element is usually determined by the spacing between it an other elements, so in general you should need to use properties like `top` and `left` and instead use `margin` for spacing between the element and outside elements and `padding` for the spacing between the element and its children.